packages/knex/src/index.ts

```diff
@@ -97,7 +97,7 @@ class DomainKnex {
           .references('id')
           .inTable(parentTableName)
           .onDelete('CASCADE');
-        debug(`Foreign key ${tableName} -> ${parentTableName}.${parentTableName}_id`);
+        debug(`Foreign key parent ${tableName} -> ${parentTableName}.${parentTableName}_id`);
```

Comment:
I disagree with this change, what does it means `Foreign key parent something -> something_id`. Please either remove this change or come up with the message that has meaning in English language